gsw_oceanographic_toolbox.c

```diff
@@ -7559,7 +7559,7 @@ gsw_pt_from_t(double sa, double t, double p, double p_ref)
         dentropy_dt     = gsw_cp0/((gsw_t0 + pt)*(1.0-0.05*(1.0 - sa/gsw_sso)));
         true_entropy_part       = gsw_entropy_part(sa,t,p);
         for (no_iter=1; no_iter <= 2; no_iter++) {
-            pt_old      = pt;
+            pt_old      = pt;temperatur
```

Comment:
Glitch here, too.

Reply:
Note to self, do not edit files with the baby on your lap.